components/camel-dynamic-router/pom.xml

```diff
@@ -40,8 +41,24 @@
       <groupId>org.apache.camel</groupId>
       <artifactId>camel-core-processor</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>jakarta.validation</groupId>
+      <artifactId>jakarta.validation-api</artifactId>
+      <version>3.0.2</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.16.0</version>
```

Comment:
Do not use hardcoded versions
And why is jackson needed ?

Reply:
I like using a jackson object mapper to get a string representation of an object, but I see how that is not really necessary, so I'll get rid of it.

Reply:
It's gone.